shepherd/Utils.py

```diff
@@ -139,3 +139,54 @@ class STATE():
     WAIT = "wait"
     TELEOP = "teleop"
     END = "end"
+
+class INGREDIENT():
+    BREAD = "Bread"
+    VEGETABLES = "Vegetables"
+    MEAT = "Meat"
+    CHEESE = "Cheese"
+    # COOKED_MEAT = "COOKED " + MEAT
+    # COOKED_BREAD = "COOKED " + BREAD
+    # COOKED_CHEESE = "COOKED " + CHEESE
+
+class RECIPIES():
```

Comment:
The word "recipes" doesn't have an 'i', BIG SAD :'(

Reply:
Shoot
I thought it would be funny with "PIE" in the name
(nah, it's the reason why I'm not a Humanities major. My engrish is horribl)